pkg/controllers/disruption/consolidation.go

```diff
@@ -35,12 +36,15 @@ import (
 	"sigs.k8s.io/karpenter/pkg/controllers/provisioning"
 	"sigs.k8s.io/karpenter/pkg/controllers/state"
 	"sigs.k8s.io/karpenter/pkg/events"
+	"sigs.k8s.io/karpenter/pkg/operator/options"
 	"sigs.k8s.io/karpenter/pkg/scheduling"
 )
 
 // consolidationTTL is the TTL between creating a consolidation command and validating that it still works.
 const consolidationTTL = 15 * time.Second
 
+var MinInstanceTypesForSpotConsolidation = 15
```

Comment:
+1, I'm assuming that the original thought here is that this would allow other providers to change this value rather than forcing them to work with the constant value that we choose but I'd rather wait for that request to come through and then change it then rather than prep for it now

Reply:
Got it. Yeah, that was the intention. I have changed it to `const`.